make-dist

```diff
@@ -63,7 +63,7 @@ download_boost() {
 
 _python_autoselect() {
   python_command=
-  for interpreter in python2.7 python3 ; do
+  for interpreter in python3 python2.7 ; do
```

Comment:
i don't have the expertise to tell what the difference between the artifacts built from a py3 nodeenv and a py2 nodeenv.

if they are the same, the change loos good to me.